Source/Core/Core/ConfigManager.cpp

```diff
@@ -226,6 +226,7 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("SelectedLanguage", SelectedLanguage);
   core->Set("OverrideGCLang", bOverrideGCLanguage);
   core->Set("DPL2Decoder", bDPL2Decoder);
+  core->Set("DPL2Quality", m_dpl2_quality);
```

Comment:
Please use the new configuration system for this instead of touching ConfigManager, which is deprecated.

In MainSettings.h, add:
```cpp
extern const ConfigInfo<AudioCommon::DPL2Quality> MAIN_DPL2_QUALITY;
```

And in MainSettings.cpp, add:
```cpp
const ConfigInfo<AudioCommon::DPL2Quality> MAIN_DPL2_QUALITY{{System::Main, "Core", "DPL2Quality"}, AudioCommon::DPL2Quality::High};
```

Finally:
```suggestion
```